components/gitpod-db/go/oidc_client_config_test.go

```diff
@@ -111,6 +111,65 @@ func TestGetOIDCClientConfigForOrganization(t *testing.T) {
 
 }
 
+func TestGetActiveOIDCClientConfigByOrgSlug(t *testing.T) {
+
+	t.Run("not found when config does not exist", func(t *testing.T) {
+		conn := dbtest.ConnectForTests(t)
+
+		_, err := db.GetActiveOIDCClientConfigByOrgSlug(context.Background(), conn, "non-existing-org")
+		require.Error(t, err)
+		require.ErrorIs(t, err, db.ErrorNotFound)
+	})
+
+	t.Run("retrieves config which exists", func(t *testing.T) {
+		conn := dbtest.ConnectForTests(t)
+
+		// create a single team
+		team := dbtest.CreateTeams(t, conn, db.Team{})[0]
+
+		created := dbtest.CreateOIDCClientConfigs(t, conn, db.OIDCClientConfig{
+			OrganizationID: team.ID,
+			Active:         true,
+		})[0]
+
+		retrieved, err := db.GetActiveOIDCClientConfigByOrgSlug(context.Background(), conn, team.Slug)
+		require.NoError(t, err)
+
+		require.Equal(t, created, retrieved)
+	})
+
+	t.Run("retrieves single active config", func(t *testing.T) {
+		conn := dbtest.ConnectForTests(t)
+
+		// create a single team
+		team := dbtest.CreateTeams(t, conn, db.Team{})[0]
+
+		// create multiple
+		activeConfigID := uuid.New()
+		configs := dbtest.CreateOIDCClientConfigs(t, conn, db.OIDCClientConfig{
+			ID:             uuid.New(),
+			OrganizationID: team.ID,
+			Active:         false,
+		}, db.OIDCClientConfig{
+			ID:             activeConfigID,
+			OrganizationID: team.ID,
+			Active:         true,
+		}, db.OIDCClientConfig{
+			ID:             uuid.New(),
+			OrganizationID: team.ID,
+			Active:         false,
+		})
+		require.Equal(t, 3, len(configs))
```

Comment:
```suggestion
		require.Len(t, configs, 3)
```
Just a suggestion, as is fine too!